scripts/postinstall.sh

```diff
@@ -3,7 +3,8 @@
 set -e
 
 cp node_modules/testpilot-metrics/testpilot-metrics.js src/vendor
-cp node_modules/kinto-http/dist/kinto-http.js src/sidebar/vendor
-cp node_modules/quill/dist/quill.js src/sidebar/vendor
+
+cp node_modules/kinto-http/dist/kinto-http.min.js src/sidebar/vendor/kinto-http.js
+cp node_modules/quill/dist/quill.min.js src/sidebar/vendor/quill.js
 cp node_modules/quill/dist/quill.snow.css src/sidebar/vendor
-cp node_modules/jose-jwe-jws/dist/jose.js src/sidebar/vendor
+cp node_modules/jose-jwe-jws/dist/jose.min.js src/sidebar/vendor/jose.js
```

Comment:
This all seemed like path of least resistance. Instead of updating all the HTML templates to use "jose.min.js", I just rename the files back to their original non-minified names during the `cp` command. 🤷‍♀️